### ISSUE TITLE
🐞  Dockerfile build support doesn't produce any outputs

### ISSUE THREAD

**Issue by marcosnils** [dagger/dagger#7113]
### What is the issue?

building a Dockerfile with dagger doesn't show any of the outputs the Dockerfile currently performs. This used to be the case before v0.11.x so it's very likely a regression on the new telemetry solution.

### Dagger version

v0.11.1

### Steps to reproduce

```Dockerfile
FROM alpine

RUN find /
```

```go
func (m *Test) Build(ctx context.Context, df *File) (*Container, error) {
	return dag.Directory().WithFile("Dockerfile", df).
		DockerBuild().Sync(ctx)
}
```

running the following function via `dagger call build --df Dockerfile` doesn't show the `find`  output the Dockerfile is currently performing 

### Log output

N/A

**Comment by jedevc** [comment]
cc @vito

**Comment by marcosnils** [comment]
closing since its fixed in v0.11.6
